dashboard/src/pages/ingest/upload/index.vue

```diff
@@ -18,11 +19,34 @@ const authStore = useAuthStore();
 const layoutStore = useLayoutStore();
 
 const GiB = 1024 ** 3; // 1 GiB in bytes
-const maxFileSize = 4 * GiB;
+const uploadMaxDefault = 1 * GiB;
+const uploadMaxSize = ref(uploadMaxDefault);
 
 layoutStore.updateBreadcrumb([{ text: "Ingest" }, { text: "Upload SIPs" }]);
 
-const uppy = new Uppy({ restrictions: { maxFileSize: maxFileSize } }).use(XHR, {
+const getUploadMaxSize = async () => {
+  return client.about
+    .aboutAbout()
+    .then((resp) => {
+      uploadMaxSize.value = resp.uploadMaxSize;
+    })
+    .catch((e) => {
+      console.error("Error fetching upload max size:", e.message);
+      uploadMaxSize.value = uploadMaxDefault;
+    });
+};
```

Comment:
What do you think **about** creating a small **about** store? Then use it from the [AboutDialog](https://github.com/artefactual-sdps/enduro/blob/main/dashboard/src/components/AboutDialog.vue) too.

Reply:
Done, see 1580b2e1bd880e319d52e03444f5082e033c19e7